tests/unit/experiments/providers/variant_sets/multi_variant_set_tests.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import collections
+import time
+import unittest
+
+from datetime import timedelta
+
+from baseplate._compat import range
+from baseplate.experiments.providers.variant_sets.multi_variant_set import MultiVariantSet
+
+logger = logging.getLogger(__name__)
+
+
+NUM_BUCKETS_DEF = 1000
```

Comment:
If `DEF` supposed to be `DEFAULT`? Can you call it that so it's more clear?

Reply:
fixed